x11-libs/libxcb/libxcb-1.13.recipe

```diff
@@ -118,29 +117,26 @@ PROVIDES_devel="
 	"
 REQUIRES_devel="
 	libxcb$secondaryArchSuffix == $portVersion base
-	devel:libxslt$secondaryArchSuffix
-	devel:libxau$secondaryArchSuffix
-	devel:libpthread_stubs$secondaryArchSuffix
 	"
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
-	devel:libxslt$secondaryArchSuffix
-	devel:xcb_proto$secondaryArchSuffix >= 1.13
-	devel:libpthread_stubs$secondaryArchSuffix
+	devel:libpthread_stubs
+	devel:util_macros
 	devel:libxau$secondaryArchSuffix
+	devel:xcb_proto
+	devel:xproto
 	"
 BUILD_PREREQUIRES="
-	devel:util_macros$secondaryArchSuffix
+	cmd:aclocal
 	cmd:autoconf
 	cmd:doxygen
+	cmd:gcc$secondaryArchSuffix
 	cmd:libtoolize$secondaryArchSuffix
-	cmd:aclocal
 	cmd:make
-	cmd:gcc$secondaryArchSuffix
-	cmd:ld$secondaryArchSuffix
 	cmd:pkg_config$secondaryArchSuffix
 	cmd:python
+	cmd:xsltproc
```

Comment:
We need a `$secondaryArchSuffix` here.
```diff
-	cmd:xsltproc
+	cmd:xsltproc$secondaryArchSuffix
```
That's because libxslt's recipe provides it with the suffix:
https://github.com/haikuports/haikuports/blob/8dd83364dfc08075d5ac885ec61e7325361c0180/dev-libs/libxslt/libxslt-1.1.32.recipe#L23

Reply:
Oops, I just noticed libxslt builds on all architectires, so the rationale in my previous comment is not correct.

The reason why we should probably add the suffix is that it will make sure we don't pull libxslt's libs for a primary arch when we build libxcb for a secondary arch.